### DISCUSSION TITLE
File shares are no longer working with Android 10 and Gennymotion 3.2.0

### DISCUSSION THREAD

**Discussion by LanThrusteR** [Genymobile/feedback#30] [Genymotion Desktop] [answered]
Works perfectly till Android 10, but starting from 10 - no file shares in /mnt/  in every Android 10 image

**Comment by jmaneyrol69** [comment, marked as answer]
Hi @LanThrusteR,

Thank you for posting. Unfortunately, we would need more information to investigate.
Please contact our technical support at https://support.genymotion.com/hc/en-us/requests/new

Thank you.
Best regards
